docs/application/administration/settings/user-interface/dashboard-and-login-screen.rst

```diff
@@ -37,7 +37,23 @@ We can select the **Dashboard Style** whether the user should see a standard **w
   * **Add OpenID Widget** quickly navigates to :doc:`../system/authentication` to configure the identity provider services (if not yet been done).
 
 
-Welcome Info and Warning
-========================
+Login Info
+==========
 
-Another option to adjust the dashboard is to add **Welcome Info** or **Welcome Warning** which is then visible above the main dashboard content (both **welcome** and **role-based**). The different between info and warning is just in the style; in both cases, we can use Markdown syntax. We should use **Welcome Info** for some generic information for users, e.g. some custom welcome message and links. On the other hand **Welcome Warning** is suitable for important notification, for example, that there is some work in progress being done or outage is planned.
+It is possible to write a message that users will see before logging in the DSW instance, using HTML or Markdown. The Login info is placed in the center of the login screen. We can use it to explain users in what cases they can/should use our DSW instance, how they should log in (e.g. if we have more :ref:`authentication services<auth-services>` configured), or if there is any news regarding our DSW instance.
+
+Sidebar Login Info
+==================
+
+It is also possible to write another message that users will see on the login screen. The Sidebar Login info is placed underneath the login form. We can also use HTML or Markdown as in the Login Info.
+
+Announcements
+=============
+
+Another option to adjust the dashboard is to add Announcements. Announcements are displayed above the main dashboard content (both **welcome** and **role-based**). There are three options:
```

Comment:
It is not just the dashboard but also the login screen (as you mentioned later).